README.md

```diff
@@ -0,0 +1,20 @@
+## Test
```

Comment:
We tend to avoid duplicating documentation on each application repo. Feel free to remove this file.

Reply:
No problem. The file has been removed.